docs/content/schedules/systemd/index.md

```diff
@@ -14,6 +14,25 @@ User systemd units are created under the user's systemd profile (`~/.config/syst
 
 System units are created in `/etc/systemd/system`
 
+## systemd drop-in files
+
+It is possible to automatically populate `*.conf.d`
+[drop-in files](https://www.freedesktop.org/software/systemd/man/latest/systemd-system.conf.html#main-conf)
+for profiles, which allows easy overriding
+of the generated services, without modifying the service templates. For example:
+
+```toml
+[root]
+inherit = "default"
+systemd-drop-in-files = "drop-in-test.conf"
```

Comment:
Could you show an example of such a file? To be honest I didn't know about that feature before you showed it to me. I suspect other users might think it's useful after seeing an full example 👍🏻

Reply:
Good point, added :)

Reply:
Nice! It gives a perfect idea on how to use the new feature 👍🏻 thanks